week-3/todo-list/script.js

```diff
@@ -17,9 +63,19 @@ function addNewTodo(event) {
   // The code below prevents the page from refreshing when we click the 'Add Todo' button.
   event.preventDefault();
   // Write your code here... and remember to reset the input field to be blank after creating a todo!
+  let newTodo = document.getElementById("todo-input").value;
+  addItem(newTodo);
 }
 
 // Advanced challenge: Write a fucntion that checks the todos in the todo list and deletes the completed ones (we can check which ones are completed by seeing if they have the line-through styling applied or not).
+
+document.getElementById("remove-all-completed").addEventListener("click", deleteAllCompletedTodos);
+
+
 function deleteAllCompletedTodos() {
-  // Write your code here...
-}
+  const doneItems = list.querySelectorAll(".done");
+  for (const item of doneItems) {
+    list.removeChild(item);
+  }
```

Comment:
All in all, some solid functional code here. It could probaly benefit from a refactor and some comments but we're not exspecting perfection here - it works that's the main thing. Good work!